packages/garbo/src/familiar/barfFamiliar.ts

```diff
@@ -203,7 +204,7 @@ export function barfFamiliar(): { familiar: Familiar; extraValue: number } {
   const fullMenu = menu({
     canChooseMacro: true,
     location: $location`Barf Mountain`,
-    includeExperienceFamiliars: false,
+    includeExperienceFamiliars: globalOptions.ascend ? false : true,
```

Comment:
```suggestion
    includeExperienceFamiliars: !globalOptions.ascend,
```

I'm also not convinced this is a great idea--right now, our experience familiar valuation is expecting _12_ experience per fight. That's not going to happen at barf, and it means they'll be significantly overvalued.